tests/flows/erc20_source_erc20_destination_multihop.go

```diff
@@ -91,7 +91,7 @@ func ERC20SourceERC20DestinationMultihop(network interfaces.Network) {
 		SecondaryFee:             big.NewInt(0),
 		RequiredGasLimit:         utils.DefaultERC20RequiredGasLimit,
 	}
-	amount := big.NewInt(0).Mul(big.NewInt(1e18), big.NewInt(13))
+	amount := new(big.Int).Mul(big.NewInt(1e18), big.NewInt(13))
```

Comment:
Just curious, is there a specific reason to construct the big int with `new`? Generally speaking, using the package's constructor is preferred, even if it's a bit clunky as is the case with the big library.

Reply:
Nope, no reason. I've preferred using `new(big.Int)` for this case personally because I think having to set a default value that doesn't get used is a little confusing, but if there's a reason why the `NewInt` constructor is preferred happy to switch.

Reply:
Looking at `NewInt`, it uses the zero values for all fields in the `x=0` case which is the same result as indirect construction via `new`.  Since that library is stable and easy to understand, I don't have a strong preference either way.

Reply:
I do think `new(big.Int)` is slightly more clear, but we do use `big.NewInt(0)` currently. I would be in favour of creating a ticket to make the switch if people prefer the former.